homeassistant/components/mqtt/discovery.py

```diff
@@ -39,6 +39,23 @@
 
 ALREADY_DISCOVERED = 'mqtt_discovered_components'
 
+ABBREVIATIONS = {
+    'avail': 'available',
+    'cmd': 'command',
+    'tmp': 'template',
+    'val': 'value',
+    'stat': ' state',
+    'pl': 'payload',
+    't': 'topic',
+    'attr': 'attributes',
+    'bri': 'brightness',
+    'opt': 'optimistic',
+    'meas': 'measurement',
+    'osc': 'oscillation',
+    'avty': 'availability',
+    'fx': 'effect',
+}
```

Comment:
The intention with 4. of https://github.com/home-assistant/architecture/issues/70 was to make the payload for the discovery smaller. Shouldn't it at least cover the required configuration variables of all platform that are supported by MQTT discovery?

Reply:
Sure, that makes sense. What is added now is the union of the 7 most used words and the 7 longest words + `effects`. I just wanted to know if the concept of abbreviated words instead of full configuration variables is OK.